src/components/Links/ContentHashLink.js

```diff
@@ -39,7 +39,7 @@ const ContentHashLink = ({ value, contentType }) => {
     return <DecodedError>{error}</DecodedError>
   }
   if (protocolType === 'ipfs') {
-    externalLink = `https://gateway.ipfs.io/ipfs/${decoded}`
+    externalLink = `https://${decoded}.ipfs.dweb.link` // using ipfs's secured origin gateway
```

Comment:
@makoto consider changing this to `https://dweb.link/ipfs/${decoded}` 

- keeps content path in URL's pathname, making it easier to copy 
- It will redirect to canonical subdomain, pointing at the same origin no matter if custom base was used (important for cookies and local storage)
- If someone uses CID that can't be represented as a subdomain, they will get useful error instead of DNS failure